src/views/css/packlink-core-override.css

```diff
@@ -0,0 +1,17 @@
+#pl-page .pl-form-group input,
+#pl-page .pl-form-group select {
+  height: 100%;
+}
+
+#pl-pricing-policy-form .pl-form-group {
+  height: 100%
+}
+
+.pl-config-list-item svg {
+  width: 22px;
+  height: 22px;
```

Comment:
This should be moved to the Core so it would be applied in all integrations.

Reply:
This was implemented before the recent Core changes (when there were svg elements instead of material icons) and is not needed anymore.